Healthy/Classes/Modules/Onboarding/Login/LoginViewController.swift

```diff
@@ -35,7 +35,6 @@ final class LoginViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
         configureAppearance()
```

Comment:
```suggestion

        configureAppearance()
```
I prefer keeping a line between view did load and configuration logic.